### PR TITLE
bug in PR#504

### PR DESCRIPTION
**Pull request by firewall2142**
The file path should've been expanded in package.ml in #504. This caused `lambdapi check natproofs.lp` to get stuck.

### CODE REVIEW

File: src/core/package.ml [hunk 1]
```diff
@@ -75,6 +75,15 @@ let read : file_path -> config_data = fun fname ->
     [Sys_error] is raised. Note that [fname] is first normalized with a call
     to [Filename.realpath]. *)
 let find_config : file_path -> file_path option = fun fname ->
+  let rec normalize fname =
```
Comment (fblanqui) [inline src/core/package.ml:78]: You already defined this function in files.ml. This should be factorized by adding a function in filename.ml.

Comment (firewall2142) [reply src/core/package.ml]: done

File: src/core/files.ml [hunk 2]
```diff
@@ -266,16 +266,7 @@ let file_to_module : string -> Path.t = fun fname ->
...
-      if Sys.file_exists fname then
-        Filename.realpath fname
-      else
-        let dirnm = Filename.dirname fname in
-        let basenm = Filename.basename fname in
-        Filename.concat (normalize dirnm) basenm
-    in
-    normalize fname
+  let fname = Filename.normalize fname
   in
```
Comment (fblanqui) [inline src/core/files.ml:270]: Thanks to put the "in" on the same line as "let" as everything holds on a single line in this case.

Comment (firewall2142) [reply src/core/files.ml]: done

**Comment by fblanqui** [comment]
Thanks @firewall2142 !

**Pull request merged**